cmd/nodecmd/create_aws.go

```diff
@@ -122,13 +131,24 @@ func getAWSCloudConfig(awsProfile string) (map[string]*awsAPI.AwsCloud, map[stri
 	}
 	for region := range finalRegions {
 		var err error
-
-		ec2SvcMap[region], err = getAWSCloudCredentials(awsProfile, region)
-		if err != nil {
-			if !strings.Contains(err.Error(), "cloud access is required") {
-				printNoCredentialsOutput(awsProfile)
+		if singleNode {
+			for _, clusterRegion := range clusterSgRegions {
```

Comment:
why do we need to pass clusterRegions as we have this info already for the cluster we work with